site/demo.html

```diff
@@ -11,26 +11,32 @@
   <div id="vis"></div>
   <script>
   var vlSpec = {
-      "data": {
-        "values": [
-          {"a":"A", "b":28}, {"a":"B", "b":55}, {"a":"C", "b":43},
-          {"a":"D", "b":91}, {"a":"E", "b":81}, {"a":"F", "b":53},
-          {"a":"G", "b":19}, {"a":"H", "b":87}, {"a":"I", "b":52}
-        ]
-      },
-      "mark": "bar",
-      "encoding": {
-        "x": {"type": "ordinal","field": "a"},
-        "y": {"type": "quantitative","field": "b"}
+    "data": {
+      "values": [
+        {"a": "C", "b": 2}, {"a": "C", "b": 7}, {"a": "C", "b": 4},
+        {"a": "D", "b": 1}, {"a": "D", "b": 2}, {"a": "D", "b": 6},
+        {"a": "E", "b": 8}, {"a": "E", "b": 4}, {"a": "E", "b": 7}
+      ]
+    },
+    "mark": "bar",
+    "encoding": {
+      "y": {"field": "a","type": "ordinal"},
+      "x": {
+        "field": "b", "type": "quantitative", "aggregate": "mean",
```

Comment:
But this is inconsistent with the rest of document.  I don't think tutorial should have priority here.

Also, you can prepend `aggregate` when you add it, which makes it totally fine.